.github/workflows/turborepo-library-release.yml

```diff
@@ -38,6 +38,29 @@ jobs:
               npm i -g pnpm@8.9.0
             setup: |
               pnpm install
+          - host: ubuntu-latest
+            container: ghcr.io/napi-rs/napi-rs/nodejs-rust:stable-2023-09-17-alpine
```

Comment:
Ah interesting, out of curiosity how did you find these?

Reply:
Copied them from [Next.js](https://github.com/vercel/next.js/blob/canary/.github/workflows/build_and_deploy.yml#L175)